gogdl/args.py

```diff
@@ -97,6 +97,7 @@ def init_parser():
         choices=["windows", "osx", "linux"],
         required=True,
     )
+    launch_parser.add_argument("--force-task", dest="force_task", default=None, help="Force playTask index to be run")
```

Comment:
maybe call this `--prefer-task` instead of `--force-task`? when I read `force` I think that it would fail if the task doesn't exist but we actually fallback to the primary task, might make it easier to understand when reading the command in the logs?

Reply:
Good point